docs/superpowers/specs/2026-05-26-upgrade-command-design.md

```
@@ -0,0 +1,176 @@
+# Upgrade Command Design
+
+## Overview
+
+Add an `antd upgrade` command that upgrades the CLI itself to the latest version by detecting the package manager used for installation and executing the corresponding upgrade command.
+
+## Command Interface
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)